tests/devapps/WAM/NetDesktopWpf/MainWindow.xaml.cs

```diff
@@ -110,22 +110,66 @@ private async void AtsAti_Click(object sender, RoutedEventArgs e)
                                      .ExecuteAsync());
 
                     result = await task.ConfigureAwait(false);
-                                     
+
                 }
                 catch (Exception ex3)
                 {
-                    DisplayMessage(ex3.ToString());
+                    DisplayMessage(ex3.Message.ToString());
+                    return;
                 }
 
             }
             catch (Exception ex2)
             {
-                DisplayMessage(ex2.ToString());
+                DisplayMessage(ex2.Message.ToString());
+                return;
             }
 
             DisplayMessage($"Success! We have a token for {result.Account.Username} valid until {result.ExpiresOn}");
         }
 
+        private async void SignOut_Click(object sender, RoutedEventArgs e)
+        {
+            var pca = CreatePublicClient();
+            var upnPrefix = UpnTbx.Text;
+
+            IEnumerable<IAccount> accounts = await pca.GetAccountsAsync().ConfigureAwait(true);
+            var acc = accounts.FirstOrDefault();
+
+            if (acc == null)
+            {
+                DisplayMessage($"No accounts to remove!!!");
+                return;
+            }
+
+            try
+            {
+                await pca.RemoveAsync(acc).ConfigureAwait(false);
+            }
+            catch (MsalException ex1)
+            {
+                try
+                {
+                    DisplayMessage(ex1.Message.ToString());
+                    return;
+
```

Comment:
Seems like there are a few extra spaces in this method

Reply:
fixed it, thanks Travis